configs/mainnet.json

```diff
@@ -483,5 +531,127 @@
+      "decimals": 18
+    },
+    "Axelar": {
+      "fee_denom": "ibc/062588846168EDFECF20F7681F2A0A131A53B9A3C8535BCA8F7D1A268246974B",
+      "decimals": 6
+    },
+    "DYDX": {
+      "fee_denom": "ibc/9DC57FF60E125F9B976EB0DF4F2122C580AD1DB8E27BA71E6D2C1AA56A5FD312",
+      "decimals": 18
+    },
+    "Osmosis": {
+      "fee_denom": "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518",
+      "decimals": 6
+    },
+    "Cosmos Hub": {
+      "fee_denom": "ibc/A4DB47A9D3CF9A068D454513891B526702455D3EF08FB9EB558C561F9DC2B701",
+      "decimals": 6
+    },
+    "Kujira": {
+      "fee_denom": "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5",
+      "decimals": 6
+    },
+    "Noble": {
+      "fee_denom": "usdc.1.6.53ff75",
+      "decimals": 6
+    },
+    "Stargaze": {
+      "fee_denom": "ibc/C688EBE80ECC20865331A18D8D19654754F6B189977715FED00FBA2BE41F144F",
+      "decimals": 6
+    },
+    "Stride": {
+      "fee_denom": "ibc/3552CECB7BCE1891DB6070D37EC6E954C972B1400141308FCD85FD148BD06DE5",
+      "decimals": 18
+    },
+    "Terra": {
+      "fee_denom": "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5",
+      "decimals": 6
+    },
+    "Mantle": {
+      "fee_denom": "bit.1.2.9d68c4",
+      "decimals": 18
+    },
+    "Base": {
+      "fee_denom": "eth.1.2.942d87",
+      "decimals": 18
+    },
+    "Evmos": {
+      "fee_denom": "ibc/35E771B8682D828173F4B795F6C307780F96DC64D6F914FAE4CC9B4666F66364",
+      "decimals": 6
+    },
+    "Juno": {
+      "fee_denom": "ibc/9E3EB38E5E157AEBFF4A8EAC66E654BC8ECFCB1F758F4D1C0F2D65945E9E2935",
+      "decimals": 6
+    },
+    "Comdex": {
+      "fee_denom": "ibc/5E60F861861B36902155FCADB0B3AD1D474376A6D4CFDBE1FC60E262016CEF7A",
+      "decimals": 6
+    },
+    "Stafihub": {
+      "fee_denom": "ibc/0249D3BAB5191A1CB96C66F76DD5B54BBBE8A8391C38691C4B980E82F385086B",
+      "decimals": 6
+    },
+    "Persistence Core": {
+      "fee_denom": "ibc/AAA241B0C1B237AA252F939C7CB7AB4C21A46BAA97464DDB8B5C92D4AF635E29",
+      "decimals": 6
+    },
+    "Canto": {
+      "fee_denom": "ibc/92E974290AF9E2BC3AEEEC35305C8FD76AC5A22A74CF8D91270FDF5A1C41E861",
+      "decimals": 18
+    },
+    "Sommelier": {
+      "fee_denom": "ibc/96BB9743B631E66525F13E5E944AF1A222AF928A4C870E457CDE932092F9528B",
+      "decimals": 18
+    },
+    "IRIS Hub": {
+      "fee_denom": "ibc/4E06CF24FEBFB3F5AF645377DCC0B70AA6183BAF6B918B8B6243FCDEB7D38118",
+      "decimals": 6
+    },
+    "Ethereum": {
+      "fee_denom": "eth.1.2.942d87",
+      "decimals": 18
+    },
+    "Arbitrum": {
+      "fee_denom": "eth.1.2.942d87",
+      "decimals": 18
+    },
+    "Polygon": {
+      "fee_denom": "brdg/e2238fb8fc5f08299c1789177d6d89bb2853238a65f6e48a9ec3b67e289f5601",
+      "decimals": 18
+    },
+    "OKC": {
+      "fee_denom": "okt.1.12.3e0516",
+      "decimals": 18
+    },
+    "Binance Smart Chain": {
+      "fee_denom": "bnb.1.6.773edb",
```

Comment:
Think the fee denom for BSC is `brdg/1768794901f8a19c2ec795a5402653cef6cbfe6b3ec6398d39fc37de963cb667`